tests/test_inference.py

```diff
@@ -0,0 +1,45 @@
+
+import os
+import random
+import pytest
+import numpy
+import torch
+from PIL import Image
+import open_clip
+import util_test
+
+os.environ['CUDA_VISIBLE_DEVICES'] = ''
+
+@pytest.mark.skip('storing and recalling of test data not ready')
```

Comment:
Why is this skipped ?

Reply:
because it needs the test data. GH actions will fail because the tests directory is not populated at this time.

Reply:
This should be removed, once it is clear how test data should be stored and at which inflection points it is to be generated.